src/Package/MSBuild.Engine.Corext/MsBuild.Engine.Corext.nuspec

```diff
@@ -14,6 +14,7 @@
     <file src="$repoRoot$/src/Package/MsBuild.Engine.Corext/README.md" target=""/>
     <file src="$repoRoot$/THIRDPARTYNOTICES.txt" target="v15.0/bin/THIRDPARTYNOTICES.txt" />
     <file src="$X86BinPath$/Microsoft.Common.props" target="\Extensions\15.0" />
```

Comment:
Add a comment like `<!-- Extensions/15.0 is deprecated. Remove this in some next version -->` :)